### PR TITLE
Add anomaly detection content

### PR DESCRIPTION
**Pull request by shaniR**

### CODE REVIEW

File: en/docs/references/alerting.md [hunk 1]
```diff
@@ -73,3 +73,64 @@ In addition, you can observe the following:
...
 
 - The status and the latency of the request (marked **4** in the image).
+
+### Anomaly alert 
+
+This alert is triggered when the current latency shows a significantly sudden upward shift compared to the average latency computed in the last 5 minutes of a resource function. Latency spikes and upward latency shifts are considered high latency. 
+
+Anomaly alerting is disabled by default. An organization administrator needs to enable it. 
+
+Choreo alerts a user of this anomaly via email. This email includes the component name, the organization the component belongs to, the environment it occurred, version, and anomaly detection information. The email has a link that takes a user to the components' latency graph in the observability view pane. A user can view the latency graph by clicking on the link **Check in Portal** in the email and and thereby [perform an RCA](https://wso2.com/choreo/docs/observability/root-cause-analysis/) for the anomaly.
```
Comment (nadheesh) [inline en/docs/references/alerting.md:83]: can't we say this includes details to identify the relevant component and the resource function anomaly time stamp etc?

File: en/docs/references/alerting.md [hunk 2]
```diff
@@ -73,3 +73,64 @@ In addition, you can observe the following:
 - The time range for which the error log applies (marked **3** in the image). However, you can scroll to view more logs that occurred during the time range selected for debugging purposes. This time range is selected via the drop-down field marked **5** in the image. You can update this time range as explained under [Select a custom time range](#select-a-custom-time-range).
 
 - The status and the latency of the request (marked **4** in the image).
+
+### Anomaly alert 
+
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/alerting.md:78]: The implementation of a resource function could include one or more endpoints right?

[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/alerting.md:78]: Used the above suggested text.

File: en/docs/references/alerting.md (hunk 1 above)
Comment (shaniR) [reply en/docs/references/alerting.md:83]: changed.

File: en/docs/references/alerting.md [hunk 3]
```diff
@@ -73,3 +73,64 @@ In addition, you can observe the following:
...
+    <div style="text-align: center; margin: 25px 0px;">
+        <a href="{{obsUrl}}" target="_blank" style="display: inline-block; border-radius: 12px; background-color: #5567d5; border: none; color: #FFFFFF; text-align: center; font-size: 16px; padding: 10px 24px; transition: all 0.5s; cursor: pointer; margin: 5px; font-weight: 700; text-decoration: none;">
+            Check in Portal
+        </a>
+    </div>
+</div>
+```
+
+#### Aggregated alert
+An aggregated alert is an alert that is issued by aggregating all anomalies detected in a 15-minute window after the immediate alert. The email format of the aggregated alert is as follows:
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/alerting.md:115]: yes you're right. removed this section.  I will add the screenshots of the emails when I receive them,

File: en/docs/references/alerting.md [hunk 4]
```diff
@@ -73,3 +73,64 @@ In addition, you can observe the following:
 - The time range for which the error log applies (marked **3** in the image). However, you can scroll to view more logs that occurred during the time range selected for debugging purposes. This time range is selected via the drop-down field marked **5** in the image. You can update this time range as explained under [Select a custom time range](#select-a-custom-time-range).
 
 - The status and the latency of the request (marked **4** in the image).
+
+### Anomaly alert 
+
+This alert is triggered when the current latency shows a significantly sudden upward shift compared to the average latency computed in the last 5 minutes of a resource function. Latency spikes and upward latency shifts are considered high latency. 
+
+Anomaly alerting is disabled by default. An organization administrator needs to enable it.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/alerting.md]: done

File: en/docs/references/alerting.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
+Alerts are notifications sent by the Choreo Alert Manager when the components that run in the production environment are not functioning as expected. Whenever a critical error (e.g., out-of-memory error) occurs, the component logs an error, and the Alert Manager notifies the members of your organization with admin rights via an email. This email contains a link to the [**Observability**](../observability/observability-overview.md) tab of the component where the metrics and logs applicable to the time interval in which the error occurred are highlighted. The Alert Publisher collects and sends alerts to Alert Manager every five minutes, which then decides when to send these alerts to the users.
```
Comment (RukshiW) [inline en/docs/references/alerting.md:3]:
```suggestion
Alerts are notifications sent by the Choreo Alert Manager when the components that run in the production environment are not functioning as expected. Whenever a critical error (e.g., an out-of-memory error) occurs, the component logs an error, and the Alert Manager notifies the members of your organization with admin rights via an email. This email contains a link to the [Observability](../observability/observability-overview.md) tab of the component where Choreo has highlighted the metrics and logs applicable to the time interval in which the error occurred. The Alert Publisher collects and sends alerts to Alert Manager every five minutes. Once the Alert Manager receives the alerts, it determines when to send them to the users.
```

File: en/docs/references/alerting.md [hunk 6]
```diff
@@ -73,3 +73,13 @@ In addition, you can observe the following:
 - The time range for which the error log applies (marked **3** in the image). However, you can scroll to view more logs that occurred during the time range selected for debugging purposes. This time range is selected via the drop-down field marked **5** in the image. You can update this time range as explained under [Select a custom time range](#select-a-custom-time-range).
 
 - The status and the latency of the request (marked **4** in the image).
+
+### Anomaly alert 
+
+This alert is triggered when average latency exhibits a considerable upward shift compared to what is observed during the last five minutes for a resource function. Latency spikes and upward latency shifts are considered high latency.
```
Comment (RukshiW) [inline en/docs/references/alerting.md:79]:
```suggestion
This alert is triggered when average latency exhibits a considerable upward shift compared to what the system observed during the last five minutes for a resource function. Latency spikes and upward latency shifts are considered high latency.
```

File: en/docs/references/alerting.md [hunk 7]
```diff
@@ -73,3 +73,13 @@ In addition, you can observe the following:
...
+
+### Anomaly alert 
+
+This alert is triggered when average latency exhibits a considerable upward shift compared to what is observed during the last five minutes for a resource function. Latency spikes and upward latency shifts are considered high latency. 
+
+**Anomaly alerting is disabled by default**. An organization administrator needs to enable it. 
+
+Choreo alerts a user of this anomaly via email. This email includes details to identify the component and the resource function, anomaly time stamp, etc. The email has a link that takes a user to the components' latency graph in the observability view pane. A user can view the latency graph by clicking on the link **Check in Portal** in the email and thereby [perform an RCA](https://wso2.com/choreo/docs/observability/root-cause-analysis/) for the anomaly. 
+
+The anomaly detector issues two types of alerts. The anomaly detector issues an immediate alert at the time the anomaly is detected and an aggregated alert that is issued by aggregating all anomalies detected in a 15-minute window following the immediate alert.
```
Comment (RukshiW) [inline en/docs/references/alerting.md:85]:
```suggestion
The anomaly detector issues two types of alerts:
The immediate alert as as soon as the anomaly is detected
The aggregated alert in which it aggregates all the anomalies detected in the last 15 minutes. The anomaly detecter sends this following the immediate alert.
```

File: en/docs/references/alerting.md [hunk 8]
```diff
@@ -50,9 +50,9 @@ If you want to change the automatically selected time interval (i.e., one hour)
 
 For more information on how to use the **Observe** tab for root cause analysis, see [Root Cause Analysis](../observability/root-cause-analysis.md).
 
-### Application Error alert
+### Application error alert
 
-This alert is triggered when you use the **“log:printError()”** function in your component and the component logs an error via that. Such errors indicate that your component is unable to function as designed, and therefore you are notified via an email so that you can troubleshoot them. The following is a sample of such a notification email. 
+This alert is triggered when you use the **“log:printError()”** function in your component and the component logs an error via that. Such errors indicate that your component is unable to function as designed, and therefore you are notified via email so that you can troubleshoot them. The following is a sample of such a notification email.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/alerting.md]: as discussed will not be addressing this change in this PR. let's open a new issue.

File: en/docs/references/alerting.md (hunk 5 above)
Comment (shaniR) [reply en/docs/references/alerting.md:3]: as discussed will not be addressing this change in this PR. let's open a new issue.

**Review by RukshiW — APPROVED** [review]

**Pull request merged**